src/Video/Renderer.hpp

```diff
@@ -19,138 +19,158 @@ namespace Video {
+
+        /// Start rendering the frame.
+        /**
+         * @param renderSurface %RenderSurface to render to.
+         */
+        VIDEO_API void StartRendering(RenderSurface* renderSurface);
+
+        /// Prepare for rendering static meshes.
+        /**
+         * @param viewMatrix The camera's view matrix.
+         * @param projectionMatrix The camera's projection matrix.
+         */
+        VIDEO_API void PrepareStaticMeshRendering(const glm::mat4& viewMatrix, const glm::mat4& projectionMatrix);
+
+        /// Render a static mesh.
+        /**
+         * @param geometry The geometry to render.
+         * @param albedo Albedo texture.
+         * @param normal Normal map.
+         * @param metallic Metallic map.
+         * @param roughness Roughness texture.
+         * @param modelMatrix Model matrix.
+         * @param isSelected Whether model is selected(should be highlighted) or not.
+         */
+        VIDEO_API void RenderStaticMesh(Geometry::Geometry3D* geometry, const Texture2D* albedo, const Texture2D* normal, const Texture2D* metallic, const Texture2D* roughness, const glm::mat4 modelMatrix, bool isSelected);
+
+        /// Update light buffer.
+        /**
+         * @param lights Vector of lights to push to the light buffer.
+         */
+        VIDEO_API void SetLights(const std::vector<Video::Light>& lights);
+
+        /// Anti-alias using FXAA.
+        /**
+         * @param renderSurface %RenderSurface to apply filter to.
+         */
+        VIDEO_API void AntiAlias(RenderSurface* renderSurface);
+
+        /// Render fog.
+        /**
+         * @param renderSurface %RenderSurface to apply filter to.
+         * @param projectionMatrix The camera's projection matrix.
+         * @param density The density of the fog.
+         * @param color Color.
+         */
+        VIDEO_API void RenderFog(RenderSurface* renderSurface, const glm::mat4& projectionMatrix, float density, const glm::vec3& color);
+
+        /// Apply glow effect.
+        /**
+         * @param renderSurface %RenderSurface to apply filter to.
+         * @param blurAmount How many times to blur the glow buffer.
+         */
+        VIDEO_API void ApplyGlow(RenderSurface* renderSurface, int blurAmount);
+
+        /// Apply a color filter.
+        /**
+         * @param renderSurface %RenderSurface to apply filter to.
+         * @param color Color.
+         */
+        VIDEO_API void ApplyColorFilter(RenderSurface* renderSurface, const glm::vec3& color);
+
+        /// Display the rendered results to back buffer.
+        /**
+         * @param renderSurface %RenderSurface to present to back buffer.
+         */
+        VIDEO_API void Present(RenderSurface* renderSurface);
+
+        /// Begin rendering icons.
+        /**
+         * Needs to be called before RenderIcon.
```

Comment:
You can use the `\note` Doxygen command here.

Reply:
Fixed

Reply:
The current solution may have the wrong syntax. See https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdnote

Reply:
Fixed